### ISSUE TITLE
dll spawn/inject doesnt work

### ISSUE THREAD

**Issue by iNoSec2** [HavocFramework/Havoc#117]
It is me again xD `dll spawn/inject` doesnt work too
_**dll inject:**_
![image](https://user-images.githubusercontent.com/51358868/193445876-9f1ca4ec-7f7e-4b63-9073-ae4ae849b157.png)

_**dll spawn**_
![image](https://user-images.githubusercontent.com/51358868/193445903-07403524-dac5-4b82-ae5a-02b24fd6eac8.png)

_**implant target**_
![image](https://user-images.githubusercontent.com/51358868/193445915-5fde1cbe-8946-4043-8a8d-3c259a5ea740.png)

**Comment by trickster0** [comment]
I have never seen a 6 digit PID lol....

**Comment by iNoSec2** [comment]
Yeah me too but it exists lol confirmes with process hacker. My personnal rdi work but nothing from Havoc

**Comment by Cracked5pider** [comment]
LMAO please type `help dll spawn` or `help dll inject` before you even try to do something. or open an issue.

**Comment by iNoSec2** [comment]
Lmao yeah i know how to use help and how to inject DLL in PID thanks @Cracked5pider

**Comment by Cracked5pider** [comment]
Yeah sorry mb. Misread it you did everything good so sorry lol. Currently taking a look into that

**Comment by iNoSec2** [comment]
Np the weird long PID got me too ;)
